.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -77,8 +77,8 @@ body:
     attributes:
       label: Packaging format used
```

Comment:
I didn't think of this, but based on your comment it makes me think the heading might not be clear in the first place.

What I meant here was more like "how are you adding Powertools to your function: using Lambda Layers or module?"

Do you have any suggestion? Or is it clear and we should just add `Other`

Reply:
I think you are right, there are only two distributions of PT NPM or Layer. I was thinking more of the bundling, how ppl package their code so it might affect the scenario of the bug. 

Let's keep it like that with only two options. We can follow up on bundling in selected cases, it wasn't crucial in the past.